terraso_backend/apps/project_management/models/projects.py

```diff
@@ -37,24 +33,19 @@ class Meta(BaseModel.Meta):
 
 
 class ProjectMembership(Membership):
-    """A proxy class created soley for graphene schema reasons"""
+    """A proxy class created solely for graphene schema reasons"""
```

Comment:
question: what are those schema reasons?

Reply:
i don't know, i was just fixing a typo in the doc comment. my guess is it creates an opaque wrapper type around `Membership` allowing us to evolve the concept of a `ProjectMembership` separately from a `LandscapeMembership`, `StoryMapMembership`, etc without needing to make breaking changes for clients